### DISCUSSION TITLE
Vercel don't display Images

### DISCUSSION THREAD

**Discussion by SelcoD** [neuefische/questions#638] [Web] [answered]
Hi, my Vercel don't display my Images. The Images at the local host get displayed.
Any Ideas ?

Localhost:

![Bildschirmfoto 2023-07-12 um 10 25 19](https://github.com/neuefische/questions/assets/132760864/83ffdf78-8b58-4ec7-814e-28079ce59f7c)

[Vercel](https://camping-friends-git-feature-detailpage-selcod.vercel.app/detailpage/1)
![Bildschirmfoto 2023-07-12 um 10 26 33](https://github.com/neuefische/questions/assets/132760864/bf0668c1-5b6a-405b-b753-044b8756c9db)

My images are in the puplic folder.

One image is a .jpg and the other one is a .jpeg.

[PR](https://github.com/SelcoD/CampingFriends/pull/6)

**Comment by SelcoD** [comment]
I changed the Images both to .jpg but it dosent work.

**Reply by JessicaLoers** [reply]
Hi @SelcoD, could you please share the error message from vercel. ✨

**Reply by SelcoD** [reply]
Hi Jessica, there is no error message

![Bildschirmfoto 2023-07-12 um 10 48 45](https://github.com/neuefische/questions/assets/132760864/20a1b8a0-3fde-4978-8558-30a6bcc2c0a8)

**Reply by SelcoD** [reply]
![Bildschirmfoto 2023-07-12 um 10 51 25](https://github.com/neuefische/questions/assets/132760864/04948b2d-5c37-4753-bc80-2837a20923f4)

**Comment by SelcoD** [comment, marked as answer]
✅ Fixed

I changed the path of the images to a relative path an it works now.

<img width="607" alt="Bildschirmfoto 2023-07-12 um 10 57 41" src="https://github.com/neuefische/questions/assets/132760864/95fb35a8-536a-44d2-b515-949ccddd5539">
